app/components/nextflow/samplesheet/metadata_cell_component.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Nextflow
+  module Samplesheet
+    # Component to render a cell with sample metadata into the sample sheet
+    class MetadataCellComponent < ViewComponent::Base
+      attr_reader :form, :metadata, :name
+
+      def initialize(form:, name:, sample:, entry:)
+        @form = form
+        @name = name
+        @metadata = if entry['meta'].is_a?(Array)
+                      sample.metadata[entry['meta'][0]].to_s
```

Comment:
**nitpick (blocking):** If the Sample does not have the metadata field requested this and line 15 will throw an error. Can you make sure that this just puts an empty string when metadata doesn't exist on the sample.

Reply:
Updated in bb21653df9057f0c5ff9dacfb3b63ec9fe79ea53